dockerfiles/Dockerfile.bookbuyer.windows

```diff
@@ -0,0 +1,5 @@
+ARG BUILD_OS=mcr.microsoft.com/windows/nanoserver/insider
+ARG BUILD_TAG=10.0.20348.1
```

Comment:
Seems like it would be possible to share a single Dockerfile for windows and linux if we override the base image from the Makefile and template the executable name. Just a thought, don't want to make things too much messier than they already are.